src/lib/MVC/Symfony/Component/Serializer/CompoundMatcherNormalizer.php

```diff
@@ -34,7 +44,24 @@ public function supportsNormalization($data, string $format = null): bool
 
     public function supportsDenormalization($data, string $type, string $format = null): bool
     {
-        return $type === Matcher\Compound::class;
+        return is_subclass_of($type, Matcher\Compound::class) || $type === Matcher\Compound::class;
```

Comment:
This should be enough, since it checks both cases?
```suggestion
        return is_a($type, Matcher\Compound::class);
```